Makefile.envs

```diff
@@ -249,7 +249,8 @@ export PYO3_CROSS_INCLUDE_DIR=${PYTHONINCLUDE}
 export RUSTFLAGS= \
 	-C link-arg=-sSIDE_MODULE=2 \
 	-C link-arg=-sWASM_BIGINT \
-	-Z link-native-libraries=no
+	-Z link-native-libraries=no \
+	--cfg pyodide_js_string
```

Comment:
is it safe to add this config even if the build backend does not accept this config?

Reply:
Yeah if the rust code doesn't check for it, it will just do nothing.